device-types/Dell/PowerEdge_R750.yaml

```diff
@@ -0,0 +1,56 @@
+---
+manufacturer: Dell
+model: PowerEdge R750
+slug: dell_poweredge_r750
+comments: 'Dell EMC PowerEdge R750 Technical Specifications https://dl.dell.com/content/manual45025655-dell-emc-poweredge-r750-technical-specifications.pdf?language=en-us&ps=true' 
+u_height: 2
+is_full_depth: true
+subdevice_role: parent
+console-ports:
+  - name: Optional Rear Serial
+    type: de-9
+  - name: Front VGA
+    type: other
+  - name: Rear VGA
+    type: other
+  - name: Internal USB1
+    type: usb-a
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front Dedicated iDRAC direct USB
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```

Comment:
We are no longer modelling USB ports unless they function solely as a console port.

The iDRAC port should be modelled as a interface as that is how it functions